src/client.ts

```diff
@@ -234,9 +267,9 @@ export class SubscriptionClient {
     }
   }
 
-  private generateRequestId() {
-    const id = this.nextRequestId;
-    this.nextRequestId += 1;
+  private generateOperationId() {
+    const id = this.nextOperationId;
```

Comment:
Yeah I prefer but it should be `return this.nextOperationId++;` to increment first and return only after :D